## This is a collection of repositories from the Cardiovascular Genomics and Precision Medicine team at Imperial College London
The Cardiovascular Genomics and Precision Medicine group (Ware lab) spans the National Heart & Lung Institute at Imperial College London, the MRC London Institute of Medical Sciences (LMS), and the Royal Brompton & Harefield Hospitals group at Guy's and St Thomas' Hospitals NHS Trust.

**Web resources from the group**  
[cvgenetics.org](https://cvgenetics.org) has information about the group and an index of our published resources  
[cardiodb.org](https://cardiodb.org) hosts our webtools  
James' [personal web page](https://www.imperial.ac.uk/people/j.ware) at Imperial College  
The [group web page](https://lms.mrc.ac.uk/research-group/cardiovascular-genomics/) at MRC LMS  


**Selected public repositories**  

|Public repos supporting published work: | PMC ID / DOI | Description |
|-|-|-|
| ACM | [PMC5957753](http://www.ncbi.nlm.nih.gov/pmc/articles/pmc5957753/) | Genetic Etiology for Alcohol-Induced Cardiac Toxicity | 
| CardioBoost_manuscript | [PMC7790749](http://www.ncbi.nlm.nih.gov/pmc/articles/pmc7790749/) | AI Missense Variant Pathogenicity Prediction for Inherited Cardiac Conditions |
| CardioClassifierManuscript | [PMC6558251](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6558251/) | CardioClassifier decision support tool for ICC variant interpretation in ACMG framework |
| frequencyFilter | [PMC5563454](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5563454/)  | Framework to use variant frequency data in clinical genome intepretation |
| homologous-missense-constraint | [medRxiv](https://doi.org/10.1101/2022.02.16.22271023) | Estimating genetic constraint at single amino acid resolution using homologous protein domain structures |
| ResponseToShahEtAl |  https://doi.org/10.1016/j.gim.2021.10.020 | A response to updated ACMG secondary findings guidelines that recommend returning SFs in cardiomyopathy genes including Titin |
| uORFs | [PMC7253449](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7253449/) | Characterising the loss-of-function impact of 5â€™ untranslated region variants in gnomAD |
| UTRannotator | [PMC8150139](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8150139/) | A VEP plug-in to annotate UTR variants |  
| ParalogueAnnotator | _pending_ | A VEP plug-in to perform paralogue annotation |  
| ParalogueAnnotation_personal | [bioRxiv](https://www.biorxiv.org/content/10.1101/2023.08.07.552236v1.full)| manuscript validating utility of paralogue annotation for missense variant interpretation |  
| Paralogue_Annotation_App | [bioRxiv](https://www.biorxiv.org/content/10.1101/2023.08.07.552236v1.full) | codebase for the paralogue annotation web app |  

|Other public repos: | |
|-|-| 
| BioinfoJournalClub | |  
| cTTN | |  
| gnomadjs | |  
| variantfx | |  
| web | |  

|Repos on [James' github](https://github.com/jamesware): | description | PMCID / DOI |
|-|-|-| 
| [alleleFrequencyApp](https://github.com/jamesware/alleleFrequencyApp) |  | PMC5563454 |  
| [denovolyzeR](https://github.com/jamesware/denovolyzeR) | An R package for statistical analyses of de novo genetic variants
| |  
| [denovolyzeR.org](https://github.com/jamesware/denovolyzeR.org) | Code for the denovolyeR website | denovolyzer.org |  
| [denovolyzerManuscript](https://github.com/jamesware/denovolyzerManuscript) | Current protocols in human genetics denovolyzeR manuscript (manuscript source code) | |  
| [denovolyzeR-ProbabilityTables](https://github.com/jamesware/denovolyzeR-ProbabilityTables) | Additional probability tables to be used with denovolyzeR | NA |  
| [eLife_Alamo2017](https://github.com/jamesware/eLife_Alamo2017) | Code to reproduce analyses presented in Alamo et al, eLife 2017 | |  
| [APPRAISE](https://github.com/jamesware/APPRAISE) | A Bayesian framework for the analysis and interpretation of rare genetic variants | PMC / URL |  
| [knitauthors](https://github.com/jamesware/knitauthors) | An unpublished R package to format & number author/affiliation lists for publication | |  
| reproSci / demo | Code for teaching / demonstrating RMarkdown, including manuscript preparation | |  
| []() | | |  






**Selected private repositories**  
An index and overview of key internal resources is maintained at https://github.com/ImperialCardioGenetics/private-github-index (accessible to team members only)
